integrations/azure/share-to-azure-functions/uni_configuration/host.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "2.0",
+  "extensionBundle": {
+    "id": "Microsoft.Azure.Functions.ExtensionBundle",
```

Comment:
I think you only need the root `host.json` file?

Reply:
Yes, 100%.  Not sure how a copy of the file snuck into the `uni_configuration` directory.